app/Http/Controllers/API/RecordsController.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace App\Http\Controllers\API;
+
+use App\Http\Controllers\Controller;
+use Request;
+
+class RecordsController extends Controller
+{
+    public function getRecords(){
+        $records = Record::all();
+
+    return response()->json( $records );
+
+    }
+
+    public function getRecord( $id ){
+        $record = Record::where('id', '=', $id)->first();
```

Comment:
This will fail when you test it because of the namespace. The controller is in the namespace `App\Http\Controllers\API` while the model doesn't have a namespace. If you access the record class with `Record`, PHP will try to access `App\Http\Controllers\API\Record` because of the namespace your controller is in. `\Record` would do the trick, but see below on how you should structure the models namespace.